cookbooks/cdo-apps/libraries/cdo_apps.rb

```diff
@@ -20,21 +20,16 @@ def setup_app(app_name)
       user user
```

Comment:
`cdo-apps` cookbook-changes summary: simplify/refactor some of the code that determines when to build `pegasus`/`dashboard`/`apps`. Previously, we had special logic that called `rake build:[dashboard|pegasus|apps]` directly. This PR simplifies this to just call `rake build` from the chef-client run whenever the Git repository is updated. This makes it easier to reason about keeping a frontend-server image up-to-date, since we only need to run `sudo chef-client` when initializing a new instance as well as when updating an existing instance with a new commit.